packages/code-du-travail-ui-old/src/Toast/taist.js

```diff
@@ -2,6 +2,7 @@ import React from "react";
 import { fireEvent, render } from "@testing-library/react";
 import Toast from "../Toast";
 
+// unusable test for now because of the button import
```

Comment:
it could be better to use `testPathIgnorePatterns`

Reply:
Next PR problem is auto solved, they will both be in the same UI. I will uncomment it, it's a promess

Reply:
done !